### PR TITLE
⭐️New: Add rule `no-duplicate-attr-inheritance`

### PR DESCRIPTION
**Pull request by privatenumber**
This rule makes sure you're setting `inheritAttrs` to `false` when it detects `v-bind="$attrs"`.

This rule is to prevent duplicated attribute inheritance: http://jsfiddle.net/hirokiosame/h0j3qrbm/

Can update the docs if there's consensus on the rule proposal.

### CODE REVIEW

**Comment by armano2** [comment]
ticket: #626

File: lib/rules/no-duplicate-attr-inheritance.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'enforce inheritAttrs: false when using v-bind="$attrs"',
+      category: 'recommended',
```
Comment (armano2) [inline lib/rules/no-duplicate-attr-inheritance.js]: Please change category to undefined
```suggestion
      category: undefined,
```

**Comment by hirokiosame** [comment]
@armano2 
What do you think of the following behavior?
- If it detects a mixin being pulled in, the rule gives the user the benefit of the doubt that `inheritAttrs: false` is set via the mixin. We can either emit a warning instead of an error, or emit no error at all.
- If the component is extended upon, then the base should be responsible for having `inheritAttrs: false` IMO. It shouldn't be the extending component's responsibility to patch a bug in the base component.

**Comment by michalsnik** [comment]
Thanks for working on it  @hirokiosame , however I'm going to close this PR for the moment, until we decide whether it's something we want to consider. Let's continue discussion in the relevant issue: #626

**Pull request closed without merging**